Dockerfile

```diff
@@ -14,6 +14,7 @@ RUN apk add --no-cache \
   libxml2-dev \
   libxslt-dev \
   tzdata \
+  git \
```

Comment:
Can be removed after we point the `webauthn` gem to a released version, if not wanted.

Reply:
Removed.